packages/x-date-pickers/src/DateTimePicker/shared.ts

```diff
@@ -59,6 +61,11 @@ export interface BaseDateTimePickerProps<TInputDate, TDate>
    * @default DateTimePickerToolbar
    */
   ToolbarComponent?: React.JSXElementConstructor<BaseToolbarProps<TDate, TDate | null>>;
+  /**
+   * Component that will replace default tabs renderer.
+   * @default DateTimePickerTabs
+   */
+  TabsComponent?: React.JSXElementConstructor<DateTimePickerTabsProps>;
   /**
```

Comment:
I've left it the same as is for the `ToolbarComponent` at the moment as they are/were closely related.
What does @flaviendelangle think about it?
Should we move these (ToolbarComponent & TabsComponent) props to `components` while we are at it?
IMHO, we should leave this part for breaking changes in `v6` 🤔

Reply:
I also think `ToolbarComponent` should become a component slot in v6 
But it would not bother me to put the tabs in the slots right away
I would be in favor to avoid future migrations whenever we can

Reply:
I've created `PickerTabs` component slot instead. 😉

Reply:
Do you think we could skip the `picker` prefix ?
We are inside a picker component to I guess it's obvious

Reply:
Fair point. 🤔 
Looking at `ActionBar`.
It simply seemed strange to have so short and generic name. 😆 
Initially I tried `TabsComponent`, but `Component` part seemed unnecessary when it's already part of `components` slots.

Reply:
I guess `Tabs` is enough :laughing:

Reply:
Done 🙈